source/common/init/manager_impl.cc

```diff
@@ -70,7 +70,7 @@ const absl::flat_hash_map<std::string, uint32_t>& ManagerImpl::unreadyTargets()
 void ManagerImpl::dumpUnreadyTargets(envoy::admin::v3::UnreadyTargetsDumps& unready_targets_dumps) {
   auto& message = *unready_targets_dumps.mutable_unready_targets_dumps()->Add();
   message.set_name(name_);
-  for (const auto& [target_name, count] : target_names_count_) {
+  for (const auto& [target_name, _] : target_names_count_) {
```

Comment:
I thought I had seen this before, and now here it is again. How does this compile? I don't think this is part of the standard so does gcc treat `_` differently?